### PR TITLE
Header section

### PR DESCRIPTION
**Pull request by 28919926**
- Add  html and css files
- Add the header section and content with corresponding styles on css.
- Fix errors on css.
- Add GitHub folder and linters files.
- Run and fix linters error.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  padding-right: 20px;
+}
+
+.headline {
+  width: 288px;
+  height: 336px;
+  left: 16px;
+  top: 104px;
+  padding: 30px 16px;
+}
```
Comment (Abenezer-Tilahun) [inline style.css]: - [x] As stated in the project requirement for "Headline section" you should use Flexbox in order to lay out the elements on the page and to make it's responsive,. Kindly Fix it.
Capture

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+     <nav class="nav">
+         <i id="menu" class="material-icons">menu</i>
+     </nav>
+    </header>
+    <section class="headline">
+        <h1 class="name">Desmond
+            Wanjala</h1>
+        <p class="p-text">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        <button type="button" class="collabo">Start collaboration</button>
```
Comment (Abenezer-Tilahun) [inline index.html:24]: - [x] As stated in the project requirement I recommend you to Implement the button interactions  styling: Active, Hover and Pressed ( Just Like the design).

Capture

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    </section>
```
Comment (Abenezer-Tilahun) [inline index.html:25]: - [x] As stated in the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly Adjust margin(bottom) for "Headline section" as defined in the design.

Your Project Look likes this : 
Capture

The Figma design Look likes this : 
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @28919926 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

- [ ] Kindly Fix the MIT License in your README, The link is broken.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,75 +1,67 @@
...
+- Technologies used
+  GITHUB, Linters and Git.
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+In order to setup a local copy of this project, dont worry all you need to do is to get started with a simple step.
+Run git clone on your git bash or VScode
```
Comment (christianbiring1) [inline README.md]: [OPTIONAL]
- [ ] It would be very nice to show how? with commands.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+<body>
+    <section class="headwrap">
+    <header class="header">
+     <nav class="nav">
+         <i id="menu" class="material-icons">menu</i>
+     </nav>
+    </header>
+    <section class="headline">
+        <h1 class="name">Desmond
+            Wanjala</h1>
```
Comment (christianbiring1) [inline index.html]: - [ ] As said by the previous reviewer, kindly make sure you stick to the design as much as possible. Your page and the template have to look similar.
Kindly make sure you have these images on your design.
Your design :point_down: 
Screenshot from 2022-10-28 08-58-28

Figma template :point_down: 
Screenshot from 2022-10-28 09-03-08

Screenshot from 2022-10-28 09-04-44

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @28919926 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

# STATUS: REQUEST CHANGES :recycle: 
soclose

 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descritpve PR `title` and `summary`

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        <p class="p-text">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        <button type="button" class="collabo">Start collaboration</button>
+        <img  class="imaghead" src="/images/headline.svg" alt="">
```
Comment (OyePriscilla) [inline index.html]: - [ ] KIndly place this image properly as the previous reviewers had noted. 😄 

## Your Design
crr13


## Figma Design 

crr12


- [ ] [optional] I am sorry that no reviewer mentioned this, the image should have been placed with CSS background property. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @28919926 😄 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. You can do it 👍 

so-close-so-close-seth-meyers

 ## Highlights
✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional README
✔️ Good commit messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @28919926, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**